Makefile

```diff
@@ -165,6 +165,12 @@ push: image
 	docker push ${REGISTRY}/yunikorn:scheduler-${VERSION}
 	docker push ${REGISTRY}/yunikorn:admission-${VERSION}
 
+#Generate the CRD code with code-generator (release-1.14)
+.PHONY: code_gen
+code_gen:
+	@echo "Generating CRD code"
+	bash ${GOPATH}/src/$(REPO)/apis/yunikorn.apache.org/update-codegen.sh
```

Comment:
We are using `go mod` to manage dependencies. That doesn't require ${GOPATH} to be set.
Can we use the relative path to this file, `pkg/apis/yunikorn.apache.org/update-codegen.sh` to run this script?
In this case, I think we will need to specify the output dir, pls see the other comment I added in the `update-codegen.sh`